src/utils/MediaUtil.ts

```diff
@@ -16,13 +16,27 @@ export class MediaUtil {
   public fetchLocalMedia = async ({
     video = false,
     audio = false,
-    videoDeviceId = "",
-    audioDeviceId = "",
   }): Promise<MediaStream> => {
     return await navigator.mediaDevices.getUserMedia({
       ...MEDIA_CONSTRAINTS,
-      video: videoDeviceId !== "" ? { deviceId: videoDeviceId } : video,
-      audio: audioDeviceId !== "" ? { deviceId: audioDeviceId } : audio,
+      video: video,
+      audio: audio,
+    });
+  };
+
+  public changeLocalVideo = async (
```

Comment:
변경하는게 아니고 얻어오는것이기 때문에 아래와 같은 함수 이름이 적절해보이네요.
```suggestion
  public fetchLocalVideo = async (
```